src/main/java/org/aksw/iguana/commons/time/TimeUtils.java

```diff
@@ -23,8 +22,23 @@ public static XSDDuration toXSDDurationInSeconds(Duration duration) {
 		return (XSDDuration) new XSDDurationType().parse("PT" + new BigDecimal(BigInteger.valueOf(duration.toNanos()), 9).toPlainString() + "S");
 	}
 
+	public static Literal createTypedDurationLiteralInSeconds(Duration duration) {
+		var seconds = "PT" + new BigDecimal(BigInteger.valueOf(duration.toNanos()), 9).toPlainString() + "S";
```

Comment:
Is it really necessary to have the seconds string here? Otherwise you could just use `duration.toString()` which also generates valid strings for XSD duration (i.e. ISO-8601).

Reply:
No, it's not necessary. I just added in, because I thought that it would make it easier to compare mutliple durations by converting them to the same time unit. But I can remove it if you want